composer.json

```diff
@@ -3,11 +3,14 @@
     "description": "Laravel plugin that helps authenticate with the auth0 service",
     "license": "MIT",
     "require": {
-        "php": ">=5.4.0",
+        "php": ">=5.5.0",
```

Comment:
I would probably bump this to 7.1 as that's going to be the required minimum with the next SDK major.